test/micro/org/openjdk/bench/java/util/TestCRC32.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021, 2022, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright line should be as follows:
 * Copyright (c) 2021,  2023, Oracle and/or its affiliates. All rights reserved.

Rest of the patch looks good to me.

Reply:
Fixed.